### ISSUE TITLE
Define standard for docstring

### ISSUE THREAD

**Issue by obiyankenobi** [HathorNetwork/hathor-core#1]
We've been using some different spacing/line-breaks for docstrings.

We need to define:
1. Should we have a space after the initial `"""`?
2. Do we ident the all lines to match the first line?

Some examples:
![image](https://user-images.githubusercontent.com/3236756/64601073-292e8780-d3b4-11e9-96b5-a2929eb32506.png)

![image](https://user-images.githubusercontent.com/3236756/64601089-3186c280-d3b4-11e9-9f87-43c905938ecd.png)

![image](https://user-images.githubusercontent.com/3236756/64601116-3ea3b180-d3b4-11e9-9f47-1a38dfc94896.png)

![image](https://user-images.githubusercontent.com/3236756/64601254-84f91080-d3b4-11e9-8501-b00a6110be1f.png)

**Comment by msbrogli** [comment]
I agree. I think we should not align all the lines to match the first. About (1), I think we can use without a space, as described in PEP-257 [1].

[1] https://www.python.org/dev/peps/pep-0257/

**Comment by obiyankenobi** [comment]
Nice reference about the PEP. We also don't seem to follow their recommendation for one-line docstrings. We have a line-break before the closing `"""`.
![image](https://user-images.githubusercontent.com/3236756/64605013-22efd980-d3bb-11e9-919e-5e8b3858bb17.png)

**Comment by jansegre** [comment]
I agree that it would be good to standardize this.

Here are some popular examples:

- django: https://github.com/django/django/blob/master/django/db/models/base.py
- asyncio: https://github.com/python/cpython/blob/master/Lib/asyncio/tasks.py
- twisted: https://github.com/twisted/twisted/blob/twisted-19.7.0/src/twisted/internet/interfaces.py

I personally like django and asyncio styles over twisted's. But I think it is more useful to be consistent than to follow any particular style.

**Comment by jansegre** [comment]
This still applies and our docstrings are still not standardized.
